ember/app/adapters/application.js

```diff
@@ -0,0 +1,29 @@
+import JSONAPIAdapter from "@ember-data/adapter/json-api";
+import OIDCAdapterMixin from "ember-simple-auth-oidc/mixins/oidc-adapter-mixin";
+
+export default class ApplicationAdapter extends JSONAPIAdapter.extend(
+  OIDCAdapterMixin
+) {
+  namespace = "api/v1";
+
+  _appendInclude(url, adapterOptions) {
+    if (adapterOptions?.include) {
+      return `${url}?include=${adapterOptions.include}`;
+    }
+    return url;
+  }
+
+  urlForUpdateRecord(id, modelName, snapshot) {
+    return this._appendInclude(
+      super.urlForUpdateRecord(id, modelName, snapshot),
+      snapshot.adapterOptions
+    );
+  }
+
+  urlForCreateRecord(modelName, snapshot) {
+    return this._appendInclude(
+      super.urlForCreateRecord(modelName, snapshot),
+      snapshot.adapterOptions
+    );
+  }
```

Comment:
I'd like a little comment explaining why we need this. Is it crucial, or just saving us an extra request?

Reply:
It saves us the extra request.

Reply:
Done

Reply:
:+1: